### DISCUSSION TITLE
Data load from folder inside S3 bucket  to POSTGRES db using PYTHON SCRIPT IN TERRAFORM AWS GLUE

### DISCUSSION THREAD

**Discussion by Sowmya-aws** [community/community#56211] [Programming Help]
### Select Topic Area

General

### Select Feature Area

- [X] Issues
- [ ] Projects

### Body

How to load data from folder inside S3 bucket  to POSTGRES db using PYTHON SCRIPT IN TERRAFORM AWS GLUE. I have written code but getting error as below:

An error occurred while calling o99.setFormat. Formats not supported for SparkSQL data sinks. Got csv

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Sowmya-aws ! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
